CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.7.1 (Sep 21, 2021)
```

Comment:
typically the version is not updated here till a new binary has been released with the updated version